src/qnsm_cfg.c

```diff
@@ -896,7 +896,7 @@ redisContext* qnsm_cfg_init_redis_ctx(void)
 
             /*auth*/
             while(NULL == (reply = redisCommand(c, cmd)));
-            QNSM_LOG(INFO, "%llu redis connect success\n", jiffies());

```

Comment:
why change this?

Reply:
> 
> 
> why change this?

QNSM_LOG default format include timestamp，so remove this duplicate code